guide/getting-started/software.md

```diff
@@ -36,66 +36,72 @@ Illustration sources
-
-{% include image.html
-   image = "/assets/images/guide/getting-started/software/cli-example.jpg"
-   retina = "/assets/images/guide/getting-started/software/cli-example@2x.jpg"
-   alt-text = "Illustrative interface of a command-line interaction"
-   caption = 'Command line interfaces are text-only. Texture by [Joel Filipe](https://unsplash.com/@joelfilip){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
-   width = 800
-   height = 350
-%}
-
-## Block explorers
+## Wallets
 
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/getting-started/software/explorer-example.jpg"
-   retina = "/assets/images/guide/getting-started/software/explorer-example@2x.jpg"
-   alt-text = "Illustrative interface for block explorer software"
-   caption = 'Block explorer offer insight into activiy on the bitcoin network. Texture by [Mike van den Bos](https://unsplash.com/@mike_van_den_bos){:target="_blank" rel="nofollow"} on [Unsplash](https://unsplash.com){:target="_blank" rel="nofollow"}.'
+   image = "/assets/images/guide/getting-started/software/wallet-example.jpg"
+   retina = "/assets/images/guide/getting-started/software/wallet-example@2x.jpg"
+   alt-text = "Illustrative interface of a wallet application"
+   caption = "Wallets allow users to send, receive, and manage transactions."
    width = 400
-   height = 388
+   height = 396
    layout = "float-right-desktop"
 %}
 
-If blockchain is a public database, block explorers are simply windows into that database. They let users view data about specific transactions, latest blocks, block height, and so on.
+Wallets are perhaps the most important Bitcoin applications. They provide easy-to-use graphical user interfaces (GUIs) designed for broad audiences that let bitcoin owners send, receive, store and manage their coins.
+
+Wallet features vary by application but always include wallet setup, balance and transaction records, and the ability to send and receive bitcoin. The full range of features that wallets may support is broad and includes security and privacy options, currency exchange features, accounting tools, interoperability, accessibility, and localization options.
 
-Explorers also provide insight into bigger picture activity on the Bitcoin network, such as daily transaction numbers. For example, there are typically fewer transactions on weekends, which results in lower fees, which are ideal for low-priority transactions.
+Few wallets support the full range of features. The reasons for this can vary from; standards not being available when the wallet was first developed (newer address formats or HD wallets for example), the choice to not include anything that implies trusting a third party, or simply because it does not fit the intended use case for the software.
```

Comment:
Adding links since the paragraph tries to explain some terms reader going through the Getting Started may not understand. 

```suggestion
Few wallets support the full range of features. The reasons for this can vary from; standards not being available when the wallet was first developed (newer [address formats]({{ '/guide/glossary/address/' | relative_url }}) or [HD wallets]({{ '/guide/glossary/wallet/#hd-wallet' | relative_url }}) for example), the choice to not include anything that implies trusting a third party, or simply because it does not fit the intended use case for the software.
```